python/lsst/ts/cbp/component.py

```diff
@@ -195,24 +195,23 @@ async def move_azimuth(self, position: float):
         else:
             await self.send_command(f"new_az={position}")
 
-    async def get_altitude(self):
+    async def get_elevation(self):
         """This gets the altitude value from the altitude encoder in degrees.
 
         Returns
         -------
         None
 
         """
-        self.log.debug("get_altitude sent")
-        self.altitude = float(await self.send_command("alt=?"))
+        self.elevation = float(await self.send_command("alt=?"))
 
-    async def move_altitude(self, position: float):
+    async def move_elevation(self, position: float):
         """This moves the vertical axis to the value that the user sent.
 
         Parameters
         ----------
         position: float
-            The value to move the altitude to which is in degrees.
+            The value to move the elevation to which is. In degrees.
```

Comment:
"to which is. In degrees" could use a bit of cleanup. Perhaps just "The desired elevation (degrees)"?